http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/DiskServiceInterp.scala

```diff
@@ -148,17 +147,16 @@ class DiskServiceInterp[F[_]: Parallel](config: PersistentDiskConfig,
       // throw 404 if no ReadPersistentDisk permission
       // Note: the general pattern is to 404 (e.g. pretend the disk doesn't exist) if the caller doesn't have
       // ReadPersistentDisk permission. We return 403 if the user can view the disk but can't perform some other action.
-      hasPermission <- authProvider.hasPersistentDiskPermission(disk.samResource,
-                                                                userInfo,
-                                                                ReadPersistentDisk,
-                                                                googleProject)
-      _ <- if (hasPermission) F.unit
+      listOfPermissions <- authProvider.getActionsWithProjectFallback(disk.samResource, googleProject, userInfo)
```

Comment:
Added the same optimizations for disks as we did for runtimes, to avoid calling Sam twice